audio-worklet/lib/param-controller.js

```diff
@@ -16,37 +16,40 @@
 class ParamController {
   /**
    * Event handler and layout for audio parameters with numeric input values
-   * @param {String} parentId id for parent element.
+   * @param {String} parentId ID for parent element.
    * @param {Function} onChangeCallback Callback to trigger on input.
-   * @param {Object} options
-   * @param {String} options.name The name of the parameter.
+   * @param {Object} options Parameters with default values if unspecified.
+   * @param {String} options.name The name of the parameter. Default is
+   *                              'Parameter'.
    * @param {String} options.id A variable name associated with the parameter.
+   *                            Defaults to the value of |name|.
    * @param {String} options.type The type of input (only support for range).
-   * @param {Number} options.min The minimum possible value.
-   * @param {Number} options.max The maximum possible value.
+   * @param {Number} options.min The minimum possible value. Default is 0.
+   * @param {Number} options.max The maximum possible value. Default is 1.
    * @param {Number} options.step The parameter's increment value.
-   * @param {Number} options.default The default value of the parameter.
+   *                              Default is 0.1.
+   * @param {Number} options.default The default value of the parameter. Default
+   *                                 is 0.
    */
   constructor(parentId, onChangeCallback, options) {
-    if (options == null) options = {};
+    if (typeof options === 'undefined') options = {};
     
     this.controller_ = document.createElement('input');
     this.name_ = options.name || 'Parameter';
     this.id_ = options.id || this.name_;
     this.controller_.type = options.type || 'range';
     this.controller_.min = options.min || 0;
 
-    if (options.max == null)
+    if (typeof options.max === 'undefined')
```

Comment:
I must have missed that suggestion, but I like it, and will change to the ternary operator.

From some quick research on typeof, it seems that the difference is that using typeof allows for the variable you are checking to not exist. In other words,` typeof f` will evaluate to true even if has never been declared. 

This doesn't matter when the variable you are checking is a member of some other object which exists (e.g. here we are checking `options.max`) and we know `options` exists. 

However, in the first use of typeof:
`if (typeof options === 'undefined') options = {};`
it does matter since, without typeof, proper default initialization won't work unless {} is given as the third argument to the ParamController, e.g.: 
`let x = new ParamController(containerId, this.setGain.bind(this), {});`
I think it is better not to force the client to give that third argument, so I will use typeof at least in 
`if (typeof options === 'undefined') options = {};`
And I think the others should be the same for symmetry and also since we really mean the more lenient version of checking if the variable is undefined (it can be both undefined and undeclared).